libraries/message/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "dora-message"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+build = false
```

Comment:
Maybe add a comment here that explains why the build script is disabled and that removing this line will re-generate the `src/message_capnp.rs` file?